models.py

```diff
@@ -0,0 +1,106 @@
+from werkzeug import generate_password_hash, check_password_hash
+from flask.ext.sqlalchemy import SQLAlchemy
+from itsdangerous import(
+    TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+# from flask.ext.api import FlaskAPI
```

Comment:
**Code Quality - Nitpick**
Commented out code should be removed completely if there's no plan of using it.